app/views/user_sessions/new.html.erb

```diff
@@ -1,22 +1,25 @@
-<h1><%= t('views.login') %></h1>
-<% if flash[:alert] %>
-  <div class="alert">
-    <%= flash[:alert] %>
+<div class='page_container'>
+  <div class='user_sessions_page_container'>
+   <h1 class='h1_user_sessions_page'><%= t('views.login') %></h1>
+   <%= form_with model: @user, url: user_sessions_path do |f| %>
+     <div class='email_text'>
```

Comment:
Let's learn the concept of BEM: https://getbem.com/
and use it to refactor where ever you can.